gen/dcos-config.yaml

```diff
@@ -211,9 +211,57 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
+          olddir /var/lib/dcos/mesos/log/archive
+          maxsize 256M
+          rotate {{ mesos_log_retention_count }}
+          copytruncate
+          postrotate
+              find /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
```

Comment:
This is supposed to delete based on when the files are last modified (anything older than 2 weeks is deleted).  We could keep it or remove it.

I can see how this might have been useful when the mesos-fetcher created a billion empty files in the agent's log directory.  But now that we're keeping a single file, outside of GLOG's control, I don't think its strictly necessary anymore.